packages/commonwealth/client/scripts/state/api/members/upgradeRoles.ts

```diff
@@ -0,0 +1,35 @@
+import { notifyError, notifySuccess } from 'controllers/app/notifications';
+import $ from 'jquery';
+import app from 'state';
+import type RoleInfo from '../../../models/RoleInfo';
+
+type UpgradeRolesProps = {
+  onRoleUpdate: (oldRole: RoleInfo, newRole: RoleInfo) => void;
+  newRole: string;
```

Comment:
type should come from here https://github.com/hicommonwealth/commonwealth/blob/fe7a3b4d01f9869135769305d38913f670a0d9b2/libs/core/src/shared-types.ts#L4-L9